Snowplow/SPTracker.m

```diff
@@ -247,6 +248,48 @@ - (void) setSessionContext:(BOOL)sessionContext {
     }
 }
 
+- (void) setGDPRContextWithBasis:(SNOWProcessingBasis)basisForProcessing
+         withDocumentDescription:(NSString *)documentDescription
+             withDocumentVersion:(NSString *)documentVersion
+                  withDocumentId:(NSString *)documentId {
+    SPPayload * pl = [[SPPayload alloc] init];
```

Comment:
Just FYI the SPPayload class is used when you want to ensure all values are strings - if you need to add values that are not strings you need to use an NSObject instead:

> https://github.com/snowplow/snowplow-objc-tracker/blob/master/Snowplow/SPSelfDescribingJson.m#L32